src/main/java/teammates/common/datatransfer/UserInfoCookie.java

```diff
@@ -50,11 +56,19 @@ public void setVerificationCode(String verificationCode) {
         this.verificationCode = verificationCode;
     }
 
+    public Instant getExpiryTime() {
+        return expiryTime;
+    }
+
+    public void setExpiryTime(Instant expiryTime) {
+        this.expiryTime = expiryTime;
```

Comment:
@wkurniawan07 Can this take in instant as an input even if we changed expiry time to a long? Thought that we would not have to validate the long this way. Or would it be more intuitive to supply a long since the field is a long?

Reply:
A setter that doesn't have the same parameter as the thing being set is more often than not problem waiting to happen. At the very least, the version that takes in the same parameter type must still be there.

Reply:
Realised that Instant.ofEpochMilli could take negative values as well as long, that give dates further than the year 2038, so no validation is required